modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -177,6 +177,8 @@ Because the following endpoints use mutual TLS authentication to establish the c
 | EU control plane: `prod-eu-central-1-starport-layer-bucket.s3.amazonaws.com` `prod-eu-central-1-starport-layer-bucket.s3.eu-central-1.amazonaws.com` | Runtime Fabric Docker image delivery.
 .2+| 443 .2+| HTTPS | US control plane: `runtime-fabric.s3.amazonaws.com` | Runtime Fabric Docker repository.
 | EU control plane: `runtime-fabric-eu.s3.amazonaws.com` | Runtime Fabric Docker repository.
+.2+| 443 .2+| HTTPS | US control plane: `configuration-resolver.prod.cloudhub.io` | Anypoint Configuration Resolver.
+| EU control plane: `configuration-resolver.prod-eu.msap.io` | Anypoint Configuration Resolver.
```

Comment:
Isn't this weird that the TLD domain is different between us and eu? 
Shouldn't eu be `configuration-resolver.prod-eu.cloudhub.io`?

Reply:
You are right that is weird but the current url is "configuration-resolver.prod-eu.msap.io" it we change it to prod-eu.cloudhub.io that will be weird as well because the transport-layer and agent reg facade are going to prod-eu.msap.io. Not sure if you want to include both for prod-eu to later analyze this change.